react/features/calendar-sync/components/MeetingList.native.js

```diff
@@ -100,10 +110,45 @@ class MeetingList extends Component<Props> {
                 disabled = { disabled }
                 onPress = { this._onPress }
                 onRefresh = { this._onRefresh }
+                renderListEmptyComponent = {
+                    this._getRenderListEmptyComponent()
```

Comment:
Is it possible to just pass the whole function in instead of its result? Maybe I keep misreading but NavigateSectionList expects renderListEmptyComponent to be a function?

Reply:
Good point, thanks, I changed it!